nix/README.md

```diff
@@ -161,6 +163,20 @@ $ nix-shell --run postgrest-style
 
 ```
 
+There is also `postgrest-style-check` that exits with a non-zero exit code if
+the check resulted in any uncommited changes. It's mostly useful for CI.
+
+## General development tools
+
+Tools like `postgrest-build`, `postgrest-run` etc. are simple wrappers around
+`cabal` and should do what you expect. `postgrest-check` runs most checks that will
+also run in CI, with the exception of the IO and Memory checks that need to be run
+separarely.
+
+`postgrest-watch` takes a command as an argument that it will rerun if any source
+file is changed. For example, `postgrest-watch postgrest-test-spec-all` will re-run
+the full spec test suite againsg all PostgreSQL version on every change.
```

Comment:
```suggestion
the full spec test suite against all PostgreSQL versions on every change.
```

Reply:
Thanks! Completed this on my phone, works surprisingly well! But it's not so great for proof-reading :)